### PR TITLE
CDA Validation

### PR DESCRIPTION
**Pull request by jonahkaye**
Refs: #[1603](https://github.com/metriport/metriport-internal/issues/1603)

### Dependencies

- Downstream: https://github.com/metriport/metriport/pull/1757

### Description

- CDA validator 

### Testing

- Local
  - [x] CDA validation working

### Release Plan

- [ ] Merge this

### CODE REVIEW

File: packages/cda-validator/server.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    }
+    res.status(500).send({ error: parseXmllintErrors(error.message) });
+  }
+});
+
+app.listen(port, () => {
+  console.log(`Server listening at http://localhost:${port}`);
+});
+
+function parseXmllintErrors(errorString: string) {
```
Comment (jonahkaye) [inline packages/cda-validator/server.ts:52]: custom error parsing logic to improve devX

File: packages/cda-validator/server.ts
Comment (jonahkaye) [inline packages/cda-validator/server.ts:1]: This is the only code I actually wrote.

File: packages/cda-validator/README.md [hunk 2]
```diff
@@ -1,21 +1,15 @@
-# cda-core-2.0
+## CDA Validator
 
-Resources for Clinical Document Architecture (CDA) release 2.0
+This is a clone of the [CDA-validator](Original repository: https://github.com/HL7/CDA-core-2.1), which contains only the schema and extensions components of the original repo.
```
Comment (leite08) [inline packages/cda-validator/README.md:3]: invalid markdown link on `CDA-validator` - it should only have one display text and one link, something like:

```
[CDA-validator](https://github.com/HL7/CDA-core-2.1)
```

or

```
CDA-validator - [original repository](https://github.com/HL7/CDA-core-2.1)
```

**Review by leite08 — APPROVED** [review]
Approving conditional to the small fix on the last comment

**Pull request merged**